resources/js/Utils/index.ts

```diff
@@ -2,6 +2,7 @@ export {itemFormatter} from './formatter'
 export {EMAIL_VALIDATION_REGEX} from './regex'
 export {DEFAULT_PAGE, DEFAULT_ROWS_PER_PAGE} from './table-constant'
 export {getFontTheme} from './theme'
+export {default as useEditDrawer} from './use-edit-drawer'
```

Comment:
don't do this, I want to import hooks in the form of `import useEditDrawer from '@Utils/use-edit-drawer'
`